test/Test/Database/LSMTree/Internal/Index/Ordinary.hs

```diff
@@ -362,7 +362,8 @@ prop_numberOfEntriesFromSerialisedIndexWorks entryCount lastKeys
     where
 
     errorMsgOrEntryCount :: Either String NumEntries
-    errorMsgOrEntryCount = fst <$> fromSBS (serialisedIndex entryCount lastKeys)
+    errorMsgOrEntryCount
+        = fst <$> fromSBS @IndexOrdinary (serialisedIndex entryCount lastKeys)
```

Comment:
If we kept the exports in the `IndexOrdinary` module, then we wouldn't have to apply a type here

Reply:
That’s right, but see [my below general comment on specialization](https://github.com/IntersectMBO/lsm-tree/pull/476#issuecomment-2491953851).